erpnext/accounts/doctype/purchase_invoice/purchase_invoice.py

```diff
@@ -673,7 +680,7 @@ def make_gle_for_rounding_adjustment(self, gl_entries):
 					"against": self.supplier,
 					"debit_in_account_currency": self.rounding_adjustment,
 					"debit": self.base_rounding_adjustment,
-					"cost_center": round_off_cost_center,
+					"cost_center": self.cost_center or round_off_cost_center,
```

Comment:
priority should be given to `round_off_cost_center `

Reply:
If user opted to tag cost center in Balance Sheet Accounts then rounded-off amount should also go to Cost Center selected in parent document otherwise trial balance wouldn't match.

Reply:
ok